docs/source/benchmarking/materials_science_and_engineering/cifcore.rst

```diff
@@ -1,13 +1,13 @@
 Crystallographic Information Framework (CIF) Core Dictionary (CIFCore)
```

Comment:
convert `Crystallographic Information Framework (CIF) Core Dictionary (CIFCore)` to `Crystallographic Information Framework Core Dictionary (CIFCore)`